lib/init/features/database.js

```diff
@@ -8,7 +8,12 @@ var utils = require('../../utils');
 var fsutils = require('../../fsutils');
 var RSVP = require('rsvp');
 
-var _getConsoleDBRules = function(instance) {
+var defaultRules = '{"rules":{".read":"auth != null",".write":"auth != null"}}';
```

Comment:
How about doing:

    JSON.stringify({rules: {'.read': 'auth != null', '.write': 'auth != null'}}, null, 2);

so that it's not all glommed on one line when then go to edit it.